ekncontent/ekncontent/eknc-xapian-bridge.c

```diff
@@ -1,19 +1,10 @@
 /* Copyright 2016 Endless Mobile, Inc. */
 
-#include "eknc-xapian-bridge.h"
+#include "eknc-xapian-bridge-private.h"
+#include "eknc-database-manager-private.h"
 
 #include <gio/gio.h>
 
-GQuark
-eknc_xapian_bridge_error_quark (void)
-{
-  return g_quark_from_static_string("eknc-xapian-bridge-error-quark");
-}
-
-#define XB_FIX_ENDPOINT "/fix"
-#define XB_QUERY_ENDPOINT "/query"
-#define XB_TEST_ENDPOINT "/test"
-
 /**
  * SECTION:xapian-bridge
```

Comment:
You have to remove this doc comment as well, and remove the `xapian-bridge` section from `ekncontent/docs/reference/ekncontent/eos-knowledge-content-docs.xml`.

Reply:
The whole file is going away anyway.